ViewStore.md

```diff
@@ -0,0 +1,135 @@
+# View Store
+
+View Store is an architecture pattern and a protocol used in SwiftUI development inspired by [The Composable Architecture](https://github.com/pointfreeco/swift-composable-architecture). A view store is an `ObservableObject` that allows us to separate view-specific logic and the rendering of a corresponding view in a way that is repeatable, prescriptive, flexible, and testable by default. It is available as a [Swift Package](https://github.com/Lickability/ViewStore).
+
+## The Protocol
+
+The protocol declaration itself is quite simple:
+
+```swift
+protocol ViewStore: ObservableObject {
+    associatedtype ViewState
+    associatedtype Action
+
+    var viewState: ViewState { get }
+    func send(_ action: Action)
+}
```

Comment:
Is this normal documented when you import the package? I'm not 100% sold on this but maybe include the docs here was well.

Reply:
> Is this normal documented when you import the package?

Yes:

```swift
// A view store allows us to separate view-specific logic and the rendering of that view in a way that is repeatable, prescriptive, flexible, and testable by default.
public protocol ViewStore: ObservableObject {

    // A container type for state associated with the corresponding view.
    associatedtype ViewState

    // Usually represented as an `enum`, `Action` represents any functionality that a view store can perform on-demand.
    associatedtype Action

    // Single source of truth for state that is used to populate a corresponding view.
    var viewState: ViewState { get }

    // Single API for the corresponding view to cause the view store perform some functionality, usually resulting in updated `viewState`.
    func send(_ action: Action)
}

```

I left them out here to not get too bogged down on details before going into them in the paragraphs that follow, but let me know if you think the docs here would be fine / not distracting up front.

Reply:
I think it is fine to leave them out.